src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -712,7 +721,7 @@ private static DataContract CreateDataContract(int id, RuntimeTypeHandle typeHan
 
                                     if (!CollectionDataContract.TryCreate(type, out dataContract))
```

Comment:
Opened https://github.com/dotnet/corefx/issues/13101 for tracking serialization of System.Reflection.Pointer